<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OnlinePlayerDialog</class>
 <widget class="QDialog" name="OnlinePlayerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>741</width>
    <height>711</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(195, 167, 112);
selection-background-color: rgb(170, 170, 127);
selection-color: rgb(170, 0, 0);

</string>
  </property>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>691</width>
     <height>651</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(195, 167, 112);
color: rgb(0, 0, 0);
border-radius: 5px;
border: 1px solid rgb(195, 167, 112);
</string>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::SizeAdjustPolicy::AdjustToContents</enum>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>689</width>
      <height>649</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>455</x>
       <y>550</y>
       <width>191</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 3px solid rgb(0, 0, 0);
background-color: rgb(195, 167, 112);
color: #333;
padding: 5px;

</string>
     </property>
     <property name="text">
      <string>Подтвердить</string>
     </property>
     <property name="autoDefault">
      <bool>true</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="emailEdit">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>480</y>
       <width>261</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid rgb(0, 0, 0);</string>
     </property>
     <property name="placeholderText">
      <string>Почта</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="registrationDateEdit">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>290</y>
       <width>261</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid rgb(0, 0, 0);</string>
     </property>
     <property name="placeholderText">
      <string>Дата регистрации</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>360</y>
       <width>261</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Активность</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="virtualNameEdit">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>410</y>
       <width>261</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid rgb(0, 0, 0);</string>
     </property>
     <property name="placeholderText">
      <string>Ник</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ageLineEdit">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>80</y>
       <width>261</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid rgb(0, 0, 0);</string>
     </property>
     <property name="placeholderText">
      <string>Возраст</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="telEdit">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>550</y>
       <width>261</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid rgb(0, 0, 0);</string>
     </property>
     <property name="placeholderText">
      <string>Телефон</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="femaleRadioButton">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>220</y>
       <width>121</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="accessibleName">
      <string>F</string>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Женский</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ratingLineEdit">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>150</y>
       <width>261</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid rgb(0, 0, 0);</string>
     </property>
     <property name="placeholderText">
      <string>Рейтинг</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="maleRadioButton">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>220</y>
       <width>121</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="accessibleName">
      <string>M</string>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Мужской</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="nameEdit">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>10</y>
       <width>261</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid rgb(0, 0, 0);</string>
     </property>
     <property name="placeholderText">
      <string>Имя</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>550</y>
       <width>61</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>(+7)</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
